src/java/org/apache/cassandra/tools/nodetool/GcStats.java

```diff
@@ -31,7 +39,69 @@ public void execute(NodeProbe probe)
         double[] stats = probe.getAndResetGCStats();
         double mean = stats[2] / stats[5];
         double stdev = Math.sqrt((stats[3] / stats[5]) - (mean * mean));
+        long[] thread = probe.numberOfGCThreads();
+
+        //value of jnaLockable
+        boolean jnaLockable = NativeLibrary.jnaMemoryLockable();
+
+        OperatingSystemMXBean osBean = (OperatingSystemMXBean) ManagementFactory.getOperatingSystemMXBean();
+
+        long freeMemoryBytes = osBean.getFreeMemorySize();
+        long swapMemoryBytes = osBean.getFreeSwapSpaceSize();
+        long totalMemoryBytes = osBean.getTotalMemorySize();
+        long totalswapMemoryBytes = osBean.getTotalSwapSpaceSize();
+
+        String PM_freeMemoryBytes = FBUtilities.prettyPrintMemory(freeMemoryBytes, " ");
```

Comment:
I prefer to use `freeMemoryBytes` without PM_` prefix here (same for all similar variables) and use e.g. `osBean.getFreeMemorySize()` call directly as the argument of `prettyPrintMemory` method.

Reply:
done!